src/battle_script_commands.c

```diff
@@ -6954,13 +6934,16 @@ static void Cmd_switchineffects(void)
 
         // Don't activate switch-in abilities if the opposing field is empty.
         // This could happen when a mon uses explosion and causes everyone to faint.
-        if (ShouldPostponeSwitchInAbilities(battler) || gBattleStruct->switchInAbilityPostponed)
+        if ((battlerAbility == ABILITY_INTIMIDATE || battlerAbility == ABILITY_DOWNLOAD)
```

Comment:
We need to make sure that when this PR makes it to upcoming, we apply the Intimidate changes to Supersweet Syrup too

Reply:
Already tried to include it until I realized this goes to master. Will make sure to take care of it.